src/styles/reset.css

```
@@ -0,0 +1,13 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+a,
+
+a:hover,
+
+a:active {
+  text-decoration: none;
```

```suggestion
a,
a:hover,
a:active {
  text-decoration: none;
```
No need to use spaces between these parameters